.circleci/orbs/orb-fsa/@orb.yml

```diff
@@ -17,21 +18,33 @@ jobs:
       api_key:
         description: Unique identifier of the organization. Can be retrieved from the admin page in your WhiteSource account.
         type: string
-        default: ""
+        default: "${API_KEY}"
       directory:
         description: Comma separated list of directories and / or files to scan.
         type: string
         default: "."
       config_file_path:
         description: Configuration file name (including file path).
         type: string
-        default: "./whitesource-fs-agent.config"
+        default: "whitesource-fs-agent.config"
```

Comment:
Should this value be updated to `wss-unified-agent.config` ?

Reply:
This is tested.. the old one wasn't properly tested ( was done by someone else)